deploy/charts/crate-operator-crds/templates/cratedbs-cloud-crate-io.yaml

```diff
@@ -173,6 +173,9 @@ spec:
                     items:
                       type: string
                     type: array
+                  allowedSystemUserCIDR:
```

Comment:
You don't need this, if the setting is coming from the environment - we don't need to be able to override it for individual CrateDBs?